packages/web-react/src/components/DropdownModern/README.md

```diff
@@ -0,0 +1,8 @@
+# DropdownModern
+
+⚠️ `DropdownModern` component is [deprecated] and will be renamed to `Dropdown` in the next major version.
+
+For more information and implementation examples, please visit the `DropdownModern` section in the [readme][dropdown] for the `Dropdown` component.
```

Comment:
Maybe we can link directly to the section?

Reply:
good idea, link added